CHANGELOG.md

```diff
@@ -24,7 +24,90 @@ Current
 - [`ResponseValidationException` uses `Response.StatusType` rather than `Response.Status`](https://github.com/yahoo/fili/pull/96)
     * `Response.StatusType` is the interface that `Response.Status` implements.
     * This will have no impact on current code in Fili that uses `ResponseValidationException`, and it allows customers to inject http
-        codes not included in `Response.Status`. 
+        codes not included in `Response.Status`.
+         
+- [Removed "provided" modifier for SLF4J and Logback dependencies in the Wikipedia example](https://github.com/yahoo/fili/pull/102)
```

Comment:
It doesn't _really_ matter. Git seems to complain less about merge conflicts if it's at the bottom I think.